libdevcore/Common.h

```diff
@@ -79,6 +79,16 @@ using bytes = std::vector<byte>;
 using bytesRef = vector_ref<byte>;
 using bytesConstRef = vector_ref<byte const>;
 
+template <class T>
+class secure_vector: public std::vector<T>
```

Comment:
We should either force this to fixed size and prevent copying or somehow invoke `cleanse` also upon copy and resize.